specification/common-types/resource-management/v1/types.json

```diff
@@ -320,6 +326,90 @@
       },
       "description": "The resource management error additional info."
     },
+    "Operation": {
+      "title": "REST API Operation",
+      "description": "Details of a REST API operation, returned from the Resource Provider Operations API",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the operation being performed on this particular object. It should match the action name that appears in RBAC / the event service. Examples: \"Microsoft.Compute/virtualMachines/write\", \"Microsoft.Compute/virtualMachines/capture/action\"",
+          "type": "string",
+          "readOnly": true
+        },
+        "isDataAction": {
+          "description": "Whether the operation applies to data-plane. This is \"true\" for data-plane operations and \"false\" for ARM/control-plane operations.",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "display": {
+          "description": "Contains the localized display information for this particular operation / action.",
+          "properties": {
+            "provider": {
+              "description": "The localized friendly form of the resource provider name, e.g. Microsoft Monitoring Insights\" or \"Microsoft Compute\"",
+              "type": "string",
+              "readOnly": true
+            },
+            "resource": {
+              "description": "The localized friendly form of the resource type related to this action/operation � it should match the public documentation for the resource provider, e.g. \"Virtual Machines\" or \"Job Schedule Collections\"",
+              "type": "string",
+              "readOnly": true
+            },
+            "operation": {
+              "description": "Required.The localized friendly name for the operation, as it should be shown to the user. It should be concise (to fit in drop downs) but clear (i.e. self-documenting), e.g. \"Create or Update Virtual Machine\", \"Restart Virtual Machine\"",
```

Comment:
The usage of "required" here isn't for end users, its required that RPs implement it.  Please remove that part.  In fact a lot of the content in these descriptions is not intended for end users (which is where this will end up).